src/main/java/tc/oc/pgm/tablist/MatchTabManager.java

```diff
@@ -94,8 +95,8 @@ public void run() {
   @Override
   public @Nullable TabView getView(Player viewer) {
     TabView view = super.getView(viewer);
-    if (view != null && !(view instanceof LegacyMatchTabView)) {
-      this.registerEvents((MatchTabView) view);
+    if (view instanceof ListeningTabView) {
```

Comment:
```suggestion
    if (view != null && view instanceof ListeningTabView) {
```

Reply:
Is a null check really needed if there is an `instanceof` check? `instanceof` will return false if the object is null.

Reply:
true, why did it have it before?